benches/rkyv.rs

```diff
@@ -0,0 +1,143 @@
+#![feature(test)]
```

Comment:
Benches are run upon commit in my dev repo, so I'm not sure if we'll blow up GitHub by adding so many new benches. We can try. 

The benchmarks are not run in the official repo which is named `main`. I named mine `master` which will trigger running benchmarks and collecting performance data upon push.